data/human/deep missions.txt

```diff
@@ -1498,91 +1594,25 @@ mission "Deep: Remnant 1C"
-		goto AC3
-				
-	label friendly1
-	`	You then tell them about the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
-		goto AC3
-	
-	label 1C2
-	branch friendly2
-		has "event: ember waste label"
-	`	The scientists look very surprised, and ask that you tell them what you found. You recall your encounter with strange-looking humans who communicate in song and sign language and attacked you after you wouldn't submit to a blood test.`
-		goto AC3
-				
-	label friendly2
-	`	The scientists look very surprised, and ask that you tell them what you found. You recall your encounter with the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
-		goto AC3
-	
-	label AC3
-	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the human beings which inhabit the region. They seem to agree that they need to learn more about these humans, and more specifically about the technology which they have developed.`
-	`	They eventually come to the conclusion that you should return to this region of space and scan the ships of these humans so that they can learn more about them.`
+conversation "deep: remnant c"
+	`The team of scientists is already waiting in the spaceport when you arrive. They remove the equipment from your ship and immediately being analyzing the results after returning to the lab.`
+	`	"At first glance this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`
+	branch revealed
+		has "deep: did reveal remnant"
+	`	"While you wait, though, I have another job you could do. You mentioned something about strange alien creatures beyond the wormhole that seemed to be able to survive in the vacuum of space. Would you be able to return there and scan them for us? These alien creatures won't have 'outifts' for the scanners to pick up, but a typical outfits scanner is likely to give us the most information."`
+		goto end
+
+	label revealed
+	`	"While you wait, though, I have another job you could do. You mentioned something about a group of humans, or at least human-like beings, beyond the wormhole. Would you be able to return there and scan their ships for us? I'd be interested in seeing what an outfits scanner returns. You need not contact them, simply find a couple ships and scan them so that we might learn more about their capabilities."`
```

Comment:
```suggestion
	`	"While you wait, though, I have another job you could do. You mentioned something about a group of humans, or at least human-like beings, beyond the wormhole. Would you be able to return there and scan their ships for us? I'd be interested in seeing what an outfits scanner returns. You need not contact them, simply find a couple of ships and scan them so that we might learn more about their capabilities."`
```